coding-standards/UI.md

```diff
@@ -0,0 +1,505 @@
+
+## Alignments
+
+Fields are aligned horizontally and vertically.  In addition to alignment, the spacing between fields both horizontally and vertically is important. 
+
+**Label Vertical alignment** – all labels must be either top aligned or center aligned. Osmosys by default uses top alignment.
+
+**Label Horizontal alignment** – all labels must be either left aligned or right aligned. Osmosys by default uses left alignment.
+
+**Label / Field Vertical spacing** - This is the space between two fields in the same column. It is very important to maintain this same across all fields in the page & application.
+
+<table style="width: 562.5px; height: 157px;">
+   <tbody>
+      <tr style="height: 31px;">
+         <td style="width: 295px; height: 31px;"></td>
+         <td style="width: 265.5px; height: 31px;">Osmosys Default
+         </td>
+      </tr>
+      <tr style="height: 31px;">
+         <td style="width: 295px; height: 31px;">Label Vertical Alignment
+         </td>
+         <td style="width: 265.5px; height: 31px;">Top
+         </td>
+      </tr>
+      <tr style="height: 31px;">
+         <td style="width: 295px; height: 31px;">Label Horizontal Alignment
+         </td>
+         <td style="width: 265.5px; height: 31px;">Left
+         </td>
+      </tr>
+      <tr style="height: 31px;">
+         <td style="width: 295px; height: 31px;">Label Font Type
+         </td>
+         <td style="width: 265.5px; height: 31px;"></td>
+      </tr>
+      <tr style="height: 31px;">
+         <td style="width: 295px; height: 31px;">Label Font Size
+         </td>
+         <td style="width: 265.5px; height: 31px;"> </td>
+      </tr>
+   </tbody>
+</table>
+
+ 
+## Sizes
+
+Proper field sizes are essential for comfortable data entry and good appearance of the user interface. Following points are to be kept in serious consideration while designing user interfaces:
+
+1. **Relevance:** Proper size for each field is important for a good user experience. An email with 50 characters shall have the field width to accommodate that size. Similarly, the field for Age can have a size that takes 3 characters. If we are  taking notes or comments, then that fields should reflect 
+2. **Uniformity:** All fields of the same type in a form / application shall have the same size. This is properly achieved when we follow a well designed CSS. The same proper sizing shall be followed even in grids.
+
+[![Picture4.jpg](https://soochna.osmosys.co/uploads/images/gallery/2016-11-Nov/scaled-840-0/Picture4.jpg)](https://soochna.osmosys.co/uploads/images/gallery/2016-11-Nov/Picture4.jpg)
+
+The above image shows three different column widths for the same data type (money) – such  things  shall  be  avoided,  unless explicitly asked by the client.
+
+## Properties
+
+This document describes the various type of fields and their properties.
+
+<table border="0" cellspacing="0" id="bkmrk-use-db-datatype-db-l">
```

Comment:
Convert this to markdown table instead. Like the above UI standards.

Reply:
Resolved in https://github.com/OsmosysSoftware/dev-standards/pull/56/commits/f0790d0686275585fd1798073a972c3e86ce88b1